erc7824-docs/docs/guides/migration-guide.md

```diff
@@ -12,6 +12,419 @@ import TabItem from '@theme/TabItem';
+});
+
+// Session key data structure
+interface RPCSessionKey {
+  id: string;
+  sessionKey: Address;
+  application: string;
+  allowances: RPCAllowanceUsage[]; // Includes usage tracking
+  scope: string;
+  expiresAt: bigint;
+  createdAt: bigint;
+}
+```
+
+#### EIP-712 Signatures: String-based Amounts
+
+EIP-712 signature types now use string values for amounts instead of numeric types to support better precision with decimal values.
+
+<Tabs>
+  <TabItem value="before" label="Before">
+
+  ```typescript
+  const types = {
+    Allowance: [
+      { name: 'asset', type: 'string' },
+      { name: 'amount', type: 'uint256' } // Numeric type
+    ]
+  };
+  ```
+
+  </TabItem>
+  <TabItem value="after" label="After">
+
+  ```typescript
+  const types = {
+    Allowance: [
+      { name: 'asset', type: 'string' },
+      { name: 'amount', type: 'string' } // String type for float amounts
+    ]
+  };
+  ```
+
+  </TabItem>
+</Tabs>
+
+### ClearNode API
+
+You should read this section only if you are using the ClearNode API directly.
+
+#### Authentication Request with Enhanced Session Keys
+
+The `auth_request` method now uses the enhanced session key parameters:
+
+<Tabs>
+  <TabItem value="application" label="Application Auth">
+
+  ```json
+  {
+    "req": [1, "auth_request", {
+      "address": "0x1234567890abcdef...",
+      "session_key": "0x9876543210fedcba...",
+      "application": "My Trading App",
+      "allowances": [
+        { "asset": "usdc", "amount": "1000.0" },
+        { "asset": "eth", "amount": "0.5" }
+      ],
+      "scope": "app.create",
+      "expires_at": 1719123456789
+    }, 1619123456789],
+    "sig": ["0x..."]
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="root" label="Root Auth (Clearnode)">
+
+  ```json
+  {
+    "req": [1, "auth_request", {
+      "address": "0x1234567890abcdef...",
+      "session_key": "0x9876543210fedcba...",
+      "application": "clearnode",
+      "allowances": [],
+      "scope": "app.create",
+      "expires_at": 1750659456789
+    }, 1619123456789],
+    "sig": ["0x..."]
+  }
+  ```
+
+  </TabItem>
+</Tabs>
+
+#### API: Zero Deposit Channel Creation
+
+The `create_channel` method no longer accepts initial deposit parameters:
+
+<Tabs>
+  <TabItem value="before" label="Before">
+
+  ```json
+  {
+    "req": [1, "create_channel", {
+      "chain_id": 137,
+      "token": "0xeeee567890abcdef...",
+      "amount": "100000000",
+      "session_key": "0x1234567890abcdef..."
+    }, 1619123456789],
+    "sig": ["0x9876fedcba..."]
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="after" label="After">
+
+  ```json
+  {
+    "req": [1, "create_channel", {
+      "chain_id": 137,
+      "token": "0xeeee567890abcdef..."
+      // amount and session_key parameters removed
+    }, 1619123456789],
+    "sig": ["0x9876fedcba..."]
+  }
+  ```
+
+  </TabItem>
+</Tabs>
+
+#### API: Session Key Management Methods
+
+New methods for session key operations have been added.
+
+**Get Session Keys Request:**
```

Comment:
This can be modified to a L5 heading as well as "Revoke Session Key Request", thus it will be easier to specify their requests and responses. We should add the response to "Revoke Session Key Request"